examples/js/misc/Timer.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+THREE.Timer = ( function () {
+
+	function Timer() {
+
+		this._previousTime = 0;
+		this._currentTime = 0;
+
+		this._deltaTime = 0;
+		this._elapsedTime = 0;
+
+		this._timeScale = 1;
```

Comment:
Nit: timescale is a word, there's no need to capitalize Scale

Reply:
👍